sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/python/PythonStreamingSinkCommitRunner.scala

```diff
@@ -39,78 +35,22 @@ import org.apache.spark.sql.types.StructType
  * from the socket, then commit or abort a microbatch.
  */
 class PythonStreamingSinkCommitRunner(
```

Comment:
After this change, is the PyhtonStreamingSinkCommitRunner the same as the batch one now?

Reply:
It is similar except that streaming commit runner also takes the batch id as parameter and throw a different type of exception.